### DISCUSSION TITLE
HELP NEEDED

### DISCUSSION THREAD

**Discussion by ah-ha-admin** [termux/termux-packages#15643] [Q&A] [answered]
![Screenshot_20230314-153402_Termux](https://user-images.githubusercontent.com/118419708/225117282-fc4c3e60-7915-4558-b2da-06c381b323fd.jpg)

**Comment by sylirre** [comment, marked as answer]
Invalid syntax in your package.json file. Open in text editor (like `nano` or `vim`) and make sure it has valid JSON syntax. You may also try doing `dos2unix package.json` to ensure the correct line endings.
